interop_binaries/Cargo.toml

```diff
@@ -30,7 +30,7 @@ futures = { version = "0.3.28", optional = true }
 fixed = { version = "1.23", optional = true }
 fixed-macro = { version = "1.1.1", optional = true }
 hex = { version = "0.4", optional = true }
-janus_aggregator_core.workspace = true
+janus_aggregator_core = { workspace = true, features = ["test-util"] }
```

Comment:
Note that this will turn on the janus_aggregator_core/test-util feature in workspace-wide builds. This shouldn't be a huge problem, as we do `-p janus_aggregator` builds as part of CI, which would catch any mistakes missed in the workspace build.

Reply:
`integration_tests/Cargo.toml` already had `janus_aggregator_core = { workspace = true, features = ["test-util"] }` so this was already true, wasn't it? In any case I agree that CI makes sure that builds without `test-util` work, as should the Docker image builds.